bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TupleValueImpl.java

```diff
@@ -390,6 +373,23 @@ public void add(long index, BString value) {
 
     // -------------------------------------------------------------------------------------------------------------
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Why we need this. change?

Reply:
It's just a refactoring since the `equal` method has been added under the constructors.